armada/handlers/armada.py

```diff
@@ -69,7 +69,7 @@ def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, _, chart, values in known_releases:
+        for chart_name, _, chart, values, _ in known_releases:
```

Comment:
so this actually a good chance to clean up the version, number var from the list, why are you ignoring status code?